src/lib/input/input-container.ts

```diff
@@ -100,10 +101,13 @@ export class MdHint {
 @Directive({
   selector: 'md-error, mat-error',
   host: {
-    'class': 'mat-input-error'
+    'class': 'mat-input-error',
```

Comment:
This class has always been here, it's used for styling to avoid having to target both element selectors